From 17fe19c1e5236731604a43ae04f4a06d48f3ec78 Mon Sep 17 00:00:00 2001
From: Roger Siddons <dizygotheca@ntlworld.com>
Date: Thu, 1 Oct 2015 20:51:11 +0100
Subject: [PATCH 07/10] MythUI: Empty menu getter

Menu construction is sometimes convoluted.
Add support to prevent empty menus being displayed.

diff --git a/mythtv/libs/libmythui/mythdialogbox.h b/mythtv/libs/libmythui/mythdialogbox.h
index 48ccaac..dc3d895 100644
--- a/mythtv/libs/libmythui/mythdialogbox.h
+++ b/mythtv/libs/libmythui/mythdialogbox.h
@@ -88,6 +88,8 @@ class MUI_PUBLIC MythMenu
 
     void SetParent(MythMenu *parent) { m_parentMenu = parent; }
 
+    bool IsEmpty()  { return m_menuItems.isEmpty(); }
+
   private:
     void Init(void);
 
-- 
2.1.4

